esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -156,6 +156,19 @@ diagnostics:
     scripts: null
 
 
+  GCP:
+    description: GCP check
+    variables:
+      nbp:
+        mip: Lmon
+      fgco2:
+        mip: Omon
+    additional_datasets:
+      - {dataset: GCP, project: OBS, tier: 2, type: reanaly, version: 2018,
+         start_year: 1959, end_year: 2017, frequency: yr}
```

Comment:
Why `frequency`?

Reply:
It's yearly data. `nbp` and `fgco2` only exist in the `Lmon` and `Omon` tables and I didn't want to introduce a new variable.

Reply:
Can you `OBS6` instead? As in `ERA5` for example.

Reply:
Same problem :confused: